locales/en.js

```diff
@@ -0,0 +1,180 @@
+export const en = {
+  welcome: {
+    title: 'Open your new bank account',
+    description_p_1: 'To open a bank account, we will need to verify your identity.',
+    description_p_2: 'It will only take a couple of minutes.',
+    next_button: 'Verify Identity'
+  },
+  document_selector: {
+    title: 'Verify your identity',
+    hint: 'Select the type of document you would like to upload',
+  },
+  upload_desktop: {
+    driving_licence: {
+      front: {
+        title: 'Front of license',
+        instructions: 'Upload front of license from your computer',
+        webcam: 'Position the front of license in the frame (it will be automatically detected)',
+      },
+      back: {
+        title: 'Back of license',
+        instructions: 'Upload back of license from your computer',
+        webcam: 'Position the back of license in the frame (it will be automatically detected)',
+      },
+    },
+    national_identity_card: {
+      front: {
+        title: 'Front of card',
+        instructions: 'Upload front of card from your computer',
+        webcam: 'Position the front of card in the frame (it will be automatically detected)',
+      },
+      back: {
+        title: 'Back of card',
+        instructions: 'Upload back of card from your computer',
+        webcam: 'Position the back of card in the frame (it will be automatically detected)',
+      },
+    },
+    passport: {
+      front: {
+        title: 'Passport photo page',
+        instructions: 'Upload passport photo page from your computer',
+        webcam: 'Position your passport photo page in the frame (it will be automatically detected)',
+      },
+    },
+    document: {
+      help: 'Having problems? Upload a file instead'
+    },
+    face: {
+      title: 'Take a selfie',
+      upload_title: 'Selfie',
+      instructions: 'Upload a selfie from your computer',
+      webcam: 'Position your face in the oval',
+      help: 'Having problems? Upload a selfie instead',
+      button: 'Take selfie',
+    }
+  },
+  upload_mobile: {
+    driving_licence: {
+      front: {
+        title: 'Front of license',
```

Comment:
redundant string

Reply:
Yes, I'm actually in the process of drying up the whole file by using interpolation